README.md

```diff
@@ -16,11 +16,13 @@
 
 ## Coding for Beginners
 - [Funny Algorithms](https://github.com/ReciHub/FunnyAlgorithms)
+- 
```

Comment:
Hey @lukeoliff I am just not sure about this line, can you fix it?

Reply:
Oh, what happened there! 😲I'll fix it!